solutions/src/main/java/ru/sberbank/school/task08/KryoSerializableManager.java

```diff
@@ -0,0 +1,73 @@
+package ru.sberbank.school.task08;
+
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import lombok.NonNull;
+import ru.sberbank.school.task08.KryoSerializerMapState;
+import ru.sberbank.school.task08.SaveGameException;
+import ru.sberbank.school.task08.SaveGameManager;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+import ru.sberbank.school.task08.state.Savable;
+import ru.sberbank.school.util.Solution;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Solution(8)
+public class KryoSerializableManager extends SaveGameManager {
+    private Kryo kryo;
+
+    public KryoSerializableManager(@NonNull String filesDirectoryPath) {
+        super(filesDirectoryPath);
+    }
+
+    @Override
+    public void initialize() {
+        kryo = new Kryo();
+        kryo.register(MapState.class, new KryoSerializerMapState());
+        kryo.register(GameObject.class);
+        kryo.register(ArrayList.class);
+        kryo.register(InstantiatableEntity.Type.class);
+        kryo.register(InstantiatableEntity.Status.class);
+    }
+
+    @Override
+    public void saveGame(String filename, Savable gameState) throws SaveGameException {
+        try(OutputStream out = new FileOutputStream(filesDirectory + filename);
+            Output output = new Output(out)){
```

Comment:
&#39;try resource&#39; has incorrect indentation level 12, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a423913448>*